manta-accounting/src/wallet/signer/nullifier_map.rs

```diff
@@ -75,6 +78,16 @@ where
         Extend::extend(self, items)
     }
 
+    #[inline]
+    fn remove(&mut self, item: &T) -> bool {
```

Comment:
btw why was the vector impl needed ?
Seems we're mostly looking up and removing, so the hashset should be better.